### PR TITLE
Get example code to compile

### PR DESCRIPTION
**Pull request by MikeHillberg**
Code for static function x:Bind wasn't compiling.

### CODE REVIEW

File: windows-apps-src/data-binding/function-bindings.md [hunk 1]
```diff
@@ -62,18 +62,19 @@ Static functions can be specified using XMLNamespace:ClassName.MethodName syntax
...
 ```csharp
-public class MyPage : Page
+static public class MyColorHelpers
 {
-    public static GenerateAppropriateForeground(SolidColorBrush background)
+    public static SolidColorBrush GenerateAppropriateForeground(Brush background)
     {
         //Implement static function
-        ...
+        return background as SolidColorBrush;
```
Comment (oldnewthing) [inline windows-apps-src/data-binding/function-bindings.md]: This is the worst possible choice for foreground, hardly an "appropriate" foreground. Consider this alternative:

    public static SolidColorBrush GenerateAppropriateForeground(Brush background)
    {
        // The only type of brush we support is solid color brushes.
        var backgroundColor = ((SolidColorBrush)background).Color;
        // Formula taken from https://www.w3.org/TR/AERT/#color-contrast
        if (backgroundColor.R * 299 + backgroundColor.G * 587 * backgroundColor.B * 114 > 500 * 255)
        {
            return new SolidColorBrush(Colors.Black);
        }
        else
        {
            return new SolidColorBrush(Colors.White);
        }
    }

Or we can avoid the problem of explaining color space theory by choosing a simpler example.

    <TextBlock x:Name="BigTextBlock" FontSize="20" Text="Big text" />
    <TextBlock FontSize="{x:Bind local:MyHelpers.Half(BigTextBlock.FontSize)}" Text="Small text" />

    static public class MyHelpers
    {
        public static double Half(double value) => value / 2.0;
    }

Comment (MikeHillberg) [reply windows-apps-src/data-binding/function-bindings.md]: Yes, I was trying to avoid color math.  I like the FontSize idea.

**Comment by garycentric** [comment]
@mcleanbyron  : The PR **passed** the acceptance checklist. Please evaluate further and approve/complete the PR if appropriate. Thanks!

**Review by mcleanbyron — APPROVED** [review]
Thanks for catching the original problem and providing a better example!

**Pull request merged**